src/image_go_nord_client/configs/messages.py

```diff
@@ -1,33 +1,27 @@
 logs = {
-
     "img": {
-        'info': "\n[INFO] Loading input image: {}",
-        'error': "\n[ERROR] On '{}': you need to pass the image path!\n\te.g. --img='Pictures/notNord.jpg'"
+        "info": "\n[INFO] Loading input image: {}",
+        "error": "\n[ERROR] On '{}': you need to pass the image path!\n\te.g. --img='Pictures/notNord.jpg'",
     },
-
     "out": {
         "info": "\n[INFO] Set output image name: {}",
-        "error": "\n[ERROR] On '{}': no output filename specify!\n\te.g. --out='Pictures/nord.jpg'"
+        "error": "\n[ERROR] On '{}': no output filename specify!\n\te.g. --out='Pictures/nord.jpg'",
     },
-
     "navg": {
         "info": "[INFO] No average pixels selected for algorithm optimization",
-        'error': "[ERROR] On '{}': the average pixels do not take any values!\n\te.g. --no-average",
+        "error": "[ERROR] On '{}': the average pixels do not take any values!\n\te.g. --no-average",
     },
-
     "pxls": [
         "[INFO] Set up pixels width area: {}",
         "[INFO] Set up pixels height area: {}",
         "[ERROR] On '{}': no value specify within the area pixels!",
```

Comment:
Do I missed something in the previous version? I think this need to be like in line 12 above:

```suggestion
        "error": "[ERROR] On '{}': no value specify within the area pixels!\n\te.g. --pixels-area=2 or -pa=-4,-3",
```

Or I've setted up like this for a reason?

Reply:
To manage this text we need to change the error parsing with a dict instead of a list. The text should be changed anyway